py/miniconf-mqtt/miniconf/miniconf.py

```diff
@@ -39,8 +39,11 @@ def __init__(self, client, prefix):
         self.prefix = prefix
         self.inflight = {}
         self.client.on_message = self._handle_response
-        self.response_topic = f'{prefix}/response'
-        self.client.subscribe(self.response_topic)
+        self.command_response_topic = f'{prefix}/response/command'
```

Comment:
The fine grained topics are because different topics require different behavior upon value reception. For example, a `list` command isn't complete after a single reception, whereas the `get` and `set` are

Reply:
Right. But that could alo be done depending on what's in `inflight`, e.g. like https://github.com/quartiq/miniconf/pull/124#discussion_r1044487541